.github/workflows/publish-to-next.yaml

```
@@ -58,3 +58,29 @@ jobs:
         env:
           HOME: ${{ github.workspace }}
           npm_config_registry: "https://fuel-024848458133.d.codeartifact.us-east-1.amazonaws.com/npm/npm/"
+
+      - uses: mshick/add-pr-comment@v2
+        with:
+          refresh-message-position: true
+          message: |
+            This PR is published in AWS CodeArtifact with version **$NPM_DIST_TAG**
+            Install using the following command:
+            ```bash
+            # PNPM
+            pnpm config set registry https://npm-packages.fuel.network/ --global
+            pnpm install fuels@$NPM_DIST_TAG
+            # NPM
+            npm --registry=https://npm-packages.fuel.network/ install fuels@$NPM_DIST_TAG
+            # Bun
+            npm_config_registry=https://npm-packages.fuel.network/ bun install fuels@$NPM_DIST_TAG
+            ```
+            Or using `create-fuels`:
```

Bit of whitespace

```suggestion
            This PR is published in AWS CodeArtifact with version **$NPM_DIST_TAG**
            
            Install using the following command:
            ```bash
            # PNPM
            pnpm config set registry https://npm-packages.fuel.network/ --global
            pnpm install fuels@$NPM_DIST_TAG
            # NPM
            npm --registry=https://npm-packages.fuel.network/ install fuels@$NPM_DIST_TAG
            # Bun
            npm_config_registry=https://npm-packages.fuel.network/ bun install fuels@$NPM_DIST_TAG
            ```
            
            Or using `create-fuels`:
```